#!/usr/bin/env bash
$PYTHON setup.py install --single-version-externally-managed --record=record.txt # solution for error: check https://github.com/conda/conda/issues/508
